### DISCUSSION TITLE
[Bug] Contributors show when clicking 'Contributors' but not as avatar in sidebar

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#49284] [Repositories]
### Select Topic Area

Bug

### Body

@XiaoYingYo contributed to my repo https://github.com/adamlui/userscripts, which shows when I click on 'Contributors'...

![image](https://user-images.githubusercontent.com/10906554/223057827-0057a5f5-331b-4bff-9007-6278508a921f.png)

However he doesn't show in the repo's sidebar (even though contributor count is accurate)

**Comment by YasinDehfuli** [comment]
Clear your browser cache and cookies: Sometimes, website data can become corrupted or outdated, which may cause issues like this to occur. Clearing your browser cache and cookies can help resolve these problems. To do this, go to your browser settings and look for an option to clear cache and cookies.

Try a different browser: Sometimes, issues can be specific to a particular browser or version. If you're experiencing the same problem in multiple browsers, skip to the next step.

Report the issue to GitHub support: If clearing your cache and cookies or trying a different browser doesn't resolve the issue, it's possible that there's a bug on GitHub's end. You can report the issue to GitHub support by filling out a form on their website. Be sure to provide as much detail as possible, including the repository URL and any error messages you're seeing.

**Reply by adamlui** [reply]
Hey it actually went away on its own (or the contributor verified his email as i advised him to) but I like your mustache
